lua/autorun/server/sv_limit_explosives.lua

```diff
@@ -0,0 +1,52 @@
+local limits = {
+    m9k_proxy = 4
+}
+
+local function getExplosivesTable(ply, class)
+    if limits[class] == nil then return end
```

Comment:
If you wanted to implement my later suggestion, this would need to turn into:
```suggestion
    if limits[class] == nil then return {} end
```

Reply:
this would break onExplosiveCreated